galaxy_ng/app/api/ui/viewsets/collection.py

```diff
@@ -168,9 +54,19 @@ def get_queryset(self):
 
     def get_serializer_class(self):
         if self.action == 'list':
-            return serializers.RepositoryCollectionListSerializer
+            return serializers.CollectionListSerializer
         else:
-            return serializers.RepositoryCollectionDetailSerializer
+            return serializers.CollectionDetailSerializer
+
+
+# TODO: Remove when ui is updated and no longer uses this endpoint
```

Comment:
This is related to the cloud deploy stuff discussed yesterday?  ie, needing to have both versions of this endpoint running until there is no ui running that needs it?